package.json

```diff
@@ -12,9 +12,6 @@
     "extension",
     "@stellar/freighter-api"
   ],
-  "resolutions": {
-    "**/terser-webpack-plugin": "^1.4.3"
```

Comment:
Webpack 5 comes with this included. This was causing an issue as this old version of Terser contained an outdated JS spec